.github/workflows/main.yml

```diff
@@ -175,3 +205,10 @@ jobs:
       with:
         name: Renders_${{ matrix.name }}
         path: build/render/
+
+    - name: Upload Shaders
```

Comment:
Now that you've added cloud-based validation of GLSL shaders to our script, I would recommend removing the more expensive uploading of shader artifacts, which are less likely to be read by developers.  Ideally I'd like to extend the pattern you've set up for GLSL, and eventually add cloud-based validation of OSL and MDL shaders.

Reply:
For now the validation (via glslang) is a syntax one only and does not actually compile the shader. Thus any errors there will not be caught. I think for shader artifacts we need a image compare against baseline but tbd how to do this.

Reply:
That's good to know, though ideally I'd like to keep the number of uploaded artifacts as small as possible, since uploads are one of the most common causes of stalls during builds.

To keep things as simple as possible, what would you think about moving all of the new generateshader.py calls to our "Python Tests" step, and removing the "Upload Shaders" step?  I think it's fine for generateshader.py to run on all targets, since it's a relatively quick step, so we likely don't need a separate flag for shader generation in our builds.